test/parallel/test-net-connect-local-error.js

```diff
@@ -3,25 +3,40 @@ const common = require('../common');
 const assert = require('assert');
 const net = require('net');
 
+function getUnassignedPort(cb) {
+  const server = net.createServer();
+  server.listen({port: 0}, common.mustCall(() => {
+    // When the server is closed this port will no longer be assigned
+    const unassignedPort = server.address().port;
+    server.close(common.mustCall(() => {
+      cb(unassignedPort);
+    }));
+  }));
+}
+
 const server = net.createServer();
-server.listen(0);
-const port = server.address().port;
-const client = net.connect({
-  port: port + 1,
-  localPort: port,
-  localAddress: common.localhostIPv4
-});
+server.listen({port: 0}, common.mustCall(() => {
+  const port = server.address().port;
+  getUnassignedPort(common.mustCall((unassignedPort) => {
+    assert.ok(unassignedPort);
+    const client = net.connect({
+      port: unassignedPort,
+      localPort: port,
+      localAddress: common.localhostIPv4
+    });
 
-client.on('error', common.mustCall(function onError(err) {
-  assert.strictEqual(
-    err.localPort,
-    port,
-    `${err.localPort} !== ${port} in ${err}`
-  );
-  assert.strictEqual(
-    err.localAddress,
-    common.localhostIPv4,
-    `${err.localAddress} !== ${common.localhostIPv4} in ${err}`
-  );
+    client.on('error', common.mustCall(function onError(err) {
+      assert.strictEqual(
+        err.localPort,
+        port,
+        `${err.localPort} !== ${port} in ${err}`
+      );
+      assert.strictEqual(
+        err.localAddress,
+        common.localhostIPv4,
+        `${err.localAddress} !== ${common.localhostIPv4} in ${err}`
+      );
+      server.close();
```

Comment:
@refack You're right. I've used `8080` like I did [here](https://github.com/nodejs/node/pull/12639) since no connection is actually made and [this issue](https://github.com/nodejs/node/issues/12376) suggests that in the future there will be a linting rule against using `common.PORT` in parallel tests.

I've changed it now, thanks!